lib/ansible/plugins/inventory/docker_swarm.py

```diff
@@ -22,9 +22,13 @@
     description:
         - Reads inventories from the Docker swarm API.
         - Uses a YAML configuration file docker_swarm.[yml|yaml].
+        - Plugin returns following groups od swarm nodes - I(all) - all hosts; I(workers) - all worker nodes;
```

Comment:
```suggestion
        - The plugin returns following groups of swarm nodes: I(all) - all hosts; I(workers) - all worker nodes;
```

Reply:
The `:` does not pass the sanity check for `ansible-doc` test so I used `-` instead. If you know how to make it pass the test let me know.

Reply:
Use quotes: ` - "The plugin returns ..."`